include/evse_security/evse_types.hpp

```diff
@@ -116,8 +116,8 @@ struct OCSPRequestDataList {
 };
 struct KeyPair {
     fs::path key;                        ///< The path of the PEM or DER encoded private key
-    fs::path certificate;                ///< The path of the PEM or DER encoded certificate
-    fs::path chain;                      ///< The path of the PEM or DER encoded certificate chain
+    fs::path certificate;                ///< The path of the PEM or DER encoded certificate chain
```

Comment:
Perhaps this can be renamed to `certificate_chain` to more clearly distinguish between `certificate_single`. This has the advantage that you will have to go through all code where `certificate` was used before, where you can verify that the chain is indeed really desired there.

Reply:
Being part of the public API, I am unsure where it might be used (maybe in non-public repos), so while I do agree with you from the clarity of the code, I would incline to keep it as-is.